src/common/math.cairo

```diff
@@ -27,3 +27,37 @@ fn mul_inverse(x: felt252) -> felt252 {
     // a ^ (p - 2) is the multiplicative inverse of a modulo p.
     pow(x, STARK_PRIME_MINUS_TWO)
 }
+
+impl Felt252Div of Div<felt252> {
```

Comment:
It seems like it has big influence on nature of felt252 type,
but if do not do this the autogenerated file will require work and will become messier.
I dont have better idea rn. I agree to go with this approach.